### PR TITLE
Feature/tooltip

### PR DESCRIPTION
**Pull request by yleaxeman**

### CODE REVIEW

File: src/icons/_icon-common-style.scss [hunk 1]
```diff
@@ -11,3 +11,9 @@
 .icon {
     @include icon-classes-generator(); // (mixin)
 }
+
```
Comment (proand) [inline src/icons/_icon-common-style.scss:14]: Why is this file touched?

**Review by proand — COMMENTED** [review]
See inline comment

File: src/icons/_icon-common-style.scss (hunk 1 above)
Comment (yleaxeman) [reply src/icons/_icon-common-style.scss]: Obsolete, now removed

**Review by proand — APPROVED** [review]
👍

**Pull request merged**